### ISSUE TITLE
Clean up pshah_customer user

### ISSUE THREAD

**Issue by gondzo** [topcoder-platform/tc-project-service#152]
pshah_customer has too many projects (more than 1024) so any query ES fails. @vikasrohit can we either delete some old projects in dev, or remove pshah_customer from some older projects?

**Comment by vikasrohit** [comment]
Yes, how do you suggest to delete those projects? Manually or through some script? Only removing from database won't work, because that won't trigger the ES update.

**Comment by coderReview** [comment]
@vikasrohit and @gondzo is this still needed? with latest changes that should not be an issue anymore.

**Comment by gondzo** [comment]
Ugh, I just checked again and total number of projects for this user is 909, so we still have some space there. Once it goes to 1024, even projects list call will fail, unless we change ES config to allow more than 1024 query terms

I guess a one time use script to remove the user from a list of projects would be enough

**Comment by coderReview** [comment]
hi @gondzo when listing project for a customer are we passing project ids to ES?

**Comment by gondzo** [comment]
yes, we have to filter projects by permission - customer and copilot can only see some of the projects https://github.com/topcoder-platform/tc-project-service/blob/1602c6c53686642b8d08dc440b1a825bb4b60e63/src/routes/projects/list.js#L323

**Comment by coderReview** [comment]
can we use this solution? https://stackoverflow.com/questions/46451920/elasticsearch-avoid-maxclausecount-by-refactoring-bool-match-clauses

**Comment by vikasrohit** [comment]
In dev we are fine with deleting projects or removing user from certain projects. However, I think we need to do some enhancements in the way we are fetching projects for customers and copilots. I think we can avoid passing project ids at least for customers because they always see the projects they are part of. And I think we can do that for copilots as well because copilots see their projects and any project which marked as reviewed and don't have a copilot yet. That means it checks project's members and project status only.
@gondzo let me know what do you think and if it seems possible, we should launch a challenge for this, not asap, but in very near future.
